drivers/note/noteram_driver.c

```diff
@@ -790,7 +803,7 @@ static int noteram_ioctl(struct file *filep, int cmd, unsigned long arg)
  *
  ****************************************************************************/
 
-void sched_note_add(FAR const void *note, size_t notelen)
+static void sched_noteram_add(FAR const void *note, size_t notelen)
```

Comment:
```suggestion
static void noteram_add(FAR struct note_driver_s *drv,
                        FAR const void *note, size_t notelen)
```
and remove line 893-902

Reply:
done